x/blob/types/payforblob.go

```diff
@@ -3,22 +3,20 @@ package types
 import (
 	"bytes"
 	"crypto/sha256"
-	math "math"
+	"math"
 
 	"github.com/celestiaorg/celestia-app/pkg/appconsts"
-	shares "github.com/celestiaorg/celestia-app/pkg/shares"
+	appshares "github.com/celestiaorg/celestia-app/pkg/shares"
```

Comment:
I don't have a strong preference either. 

My impression is that it is preferable to use the package name by default. If there is a conflict between two package names, then ideally we use a consistent import alias but that doesn't seem easily enforceable with linters so it's kinda at code maintainers discretion. In this case, we could remove the package alias by:

1. Remove the package alias and duplicate package import
```golang
import (
	"github.com/celestiaorg/celestia-app/pkg/shares"
)
```

2. Rename the variable `shares` => `blobShares` to avoid conflicting with the import name
```golang
func CreateCommitment(namespace []byte, blobData []byte, shareVersion uint8) ([]byte, error) {
	blob := coretypes.Blob{
		NamespaceID:  namespace,
		Data:         blobData,
		ShareVersion: shareVersion,
	}

	// split into shares that are length delimited and include the namespace in
	// each share
	blobShares, err := shares.SplitBlobs(0, nil, []coretypes.Blob{blob}, false)
	if err != nil {
		return nil, err
	}

	// the commitment is the root of a merkle mountain range with max tree size
	// equal to the minimum square size the blob can be included in. See
	// https://github.com/celestiaorg/celestia-app/blob/fbfbf111bcaa056e53b0bc54d327587dee11a945/docs/architecture/adr-008-blocksize-independent-commitment.md
	minSquareSize := BlobMinSquareSize(len(blobData))
	treeSizes := merkleMountainRangeSizes(uint64(len(blobShares)), uint64(minSquareSize))
	leafSets := make([][][]byte, len(treeSizes))
	cursor := uint64(0)
	for i, treeSize := range treeSizes {
		leafSets[i] = shares.ToBytes(blobShares[cursor : cursor+treeSize])
		cursor = cursor + treeSize
	}
```